EnglishFilter/sections/general_extensions.txt

```diff
@@ -163,7 +163,7 @@ nyafilmer1.com,eririo.club,upload-4ever.com,dlpsgame.*,allkindonline.com,123free
 ! `#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")`
-streamtape.com#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")
+4anime.to,streamtape.com#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZGVk|break;case \$\./")
```

Comment:
why new rule? Is `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")` not working?

Reply:
It's working, but @Yuki2718 Suggested it. I agree on both. @Yuki2718 What do you suggest?

Reply:
Sorry, it's my bad.

Reply:
Sure.